text/0026-rest-api/rest-api.yaml

```diff
@@ -1668,6 +1729,49 @@ components:
         - index
         - data
 
+
+
+
+
+    TreasuryTransactionPayload:
+      properties:
+        type:
+          type: integer
+          description: Set to value 4 to denote a Treasury Payload.
+        input:
+          allOf:
+            - $ref: '#/components/schemas/TreasuryInput'
+        output:
+          allOf:
+            - $ref: '#/components/schemas/TreasuryOutput'
+      required:
+        - type
+        - input
+        - output
+
+    TreasuryInput:
+      properties:
+        type:
+          type: integer
+          description: Set to value 1 to denote a TreasuryInput.
+        messageId:
```

Comment:
Should be milestoneHash

https://github.com/iotaledger/iota.go/blob/555f6dae0fe0faa9c7a3a5e90cf7561a17abaff9/treasury_input.go#L64